plugins/node/instrumentation-runtime-node/README.md

```diff
@@ -32,7 +32,7 @@ const prometheusExporter = new PrometheusExporter({
 const sdk = new NodeSDK({
   metricReader: prometheusExporter,
   instrumentations: [new RuntimeNodeInstrumentation({
-    eventLoopUtilizationMeasurementInterval: 5000,
+    monitoringPrecision: 5000,
```

Comment:
minor Q: It looks like the default is *10* ms (from lower down in this README). Should the example here show 10 rather than 5000?